transport/rif_comms/layer.py

```diff
@@ -44,7 +46,27 @@ def construct_light_clients_nodes(self, config):
             sys.exit(1)
 
     def light_client_onboarding_data(self, address: Address) -> dict:
-        pass
+        return {
+            "transport_mode": self.transport_type,
+        }
 
     def register_light_client(self, raiden_api: 'RaidenAPI', registration_data: dict) -> TransportNode:
-        pass
+        config = raiden_api.raiden.config["transport"]["rif_comms"]
+        address = bytearray.fromhex(remove_0x_prefix(registration_data['address']))
```

Comment:
i think it's better to explicitly call this `lc_address` to avoid confusion

Reply:
done in next commit